src/inspector_agent.cc

```diff
@@ -778,6 +778,13 @@ bool Agent::Start(const std::string& path,
     StartDebugSignalHandler();
   }
 
+  AtExit(parent_env_, [](void* env) {
```

Comment:
IIUC this is the first time we use `AtExit` internally? (I vaguely remember we have done that before but I could not find any existing AtExit calls ATM)

Reply:
Yes, I think so :+1: